src/modal/modal-container.component.ts

```diff
@@ -135,7 +135,7 @@ export class ModalContainerComponent implements OnInit, OnDestroy {
   }
 
   hide(): void {
-    if (this.isModalHiding || !this.isShown) {
+    if (this.isModalHiding) {
```

Comment:
It should hide modal even if it is not shown, as shown is set after transition delay. We could keep that if we refactor the way this is set.